src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/DefaultCompressPlugin.java

```diff
@@ -102,6 +102,13 @@ public void configure(Map<String, String> config) {
                     zip = new ZipPlugin(resFilter);
```

Comment:
For old values (0, 1, 2), jlink should emit a warning message that that value is deprecated and will be removed in a future release.